faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -714,6 +714,19 @@ public static Expr At(Expr timestamp, Expr expr) {
     return Fn.apply("at", timestamp, "expr", expr);
   }
 
+  /**
+   * Execute the reads associated with the given expression at the timestamp provided.
+   * Writes are still executed at the current transaction time.
+   *
+   * @param timestamp the read timestamp.
+   * @param expr the scoped expression
```

Comment:
We were asked to mention the type of the `Expr` parameters in the parameter description. See the function above.

Reply:
Nevermind. It doesn't make sense in this case.